src/config/entities/postgres.config.ts

```diff
@@ -37,7 +40,7 @@ export const postgresConfig = (
     password: postgresEnvConfig.password,
     database: postgresEnvConfig.database,
     migrations: ['dist/migrations/*.js'],
-    // logging: !isCIContext,
+    logger: logger ? new PostgresqlLogger(logger, undefined) : undefined,
```

Comment:
Nit: the `undefined` second parameter is redundant, isn't it?

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/1988/commits/378fdd1d0c485f4b3dbdba8a8ed0efc33d8a2121